.github/workflows/integrationtest.yml

```diff
@@ -36,6 +36,7 @@ jobs:
     if: |
       github.event_name == 'push' ||
       ( github.event_name == 'pull_request'  && contains(github.event.pull_request.labels.*.name, 'integration' ))
+      
```

Comment:
Really? That's the difference? I'm starting to hate yaml now

Reply:
I'm not sure if it is really a yaml issue. I think is more likely the parser reading this file is a bit over-sensitive to newlines and stuff. I guess you need a newline to correctly close the 
 multiline conditional.

Reply:
also, I changed the indentation... I don't know, if you used `<tab>` for indenting or spaces. I use spaces and only after re-indenting, my editor correctly understood what to highlight.